Sources/MapboxNavigation/LaneView.swift

```diff
@@ -336,24 +336,37 @@ extension ManeuverDirection {
 
 /// :nodoc:
 open class LaneView: UIView {
```

Comment:
Thanks for documenting all the members of this class! Note that they still won’t show up in jazzy because the class itself is undocumented. I think that’s OK but just FYI.